roles/ansible-test/tasks/main.yaml

```diff
@@ -33,4 +33,13 @@
     chdir: "{{ _test_location }}"
     executable: /bin/bash
   environment: "{{ ansible_test_environment | default({}) }}"
-  shell: "source {{ ansible_test_venv_path }}/bin/activate; {{ ansible_test_executable }} {{ ansible_test_test_command }} {{ ansible_test_options }} --python {{ ansible_test_python }} -vvvv {{ _integration_targets }}"
+  shell: "source {{ ansible_test_venv_path }}/bin/activate; {{ ansible_test_executable }} {{ ansible_test_test_command }} {{ ansible_test_options }} --python {{ ansible_test_python }} -vvvv {{ _integration_targets }}; {{ ansible_coverage_report }}"
+
+- name: Ensure controller directory exists
```

Comment:
same, I would like to see us create a new post-run playbook for our unit test jobs